pkg/controller/watch/handler/handler.go

```diff
@@ -71,13 +69,6 @@ func (r *Handler) Match(object meta.Object, ref core.ObjectReference) bool {
 		ref.Name == object.GetName()
 }
 
-//
-// Inventory watch has parity.
-func (r *Handler) HasParity() bool {
-	return r.parity
-
-}
```

Comment:
No longer need to keep track of when parity is reached.  That watch is created with snapshot=false.